### DISCUSSION TITLE
Implement Existing Assets for Food & Drinks

### DISCUSSION THREAD

**Discussion by jasper-one** [EveronLife/EveronLife#141] [Ideas]
**Existing Models for Food**
![image](https://user-images.githubusercontent.com/106525617/177146194-a07e4745-b05a-447a-bd3b-1da883685ef9.png)
![image](https://user-images.githubusercontent.com/106525617/177146234-98d55dbc-57e7-4f34-b8fd-3a8ea6b4d9c8.png)

**Existing Models for Drinks**
![image](https://user-images.githubusercontent.com/106525617/177146507-e20f8a83-82e6-4db3-bac2-1376da233ecb.png)

There are also some other drinks and canned foods under the "Tools" models.

**Comment by Arkensor** [comment]
Have a look at the apple juice processor, i think there is a beer bottle etc that could also be reskinned
